src/azure-cli/azure/cli/command_modules/apim/commands.py

```diff
@@ -83,6 +95,14 @@ def load_command_table(self, _):
         g.custom_command('add', 'apim_product_api_add')
         g.custom_command('delete', 'apim_product_api_delete')
 
+    with self.command_group('apim api schema', api_schema) as g:
+        g.custom_command('create', 'apim_api_schema_create', supports_no_wait=True)
+        g.custom_command('delete', 'apim_api_schema_delete', supports_no_wait=True)
```

Comment:
Do we need to consider adding ` confirmation=True` for the delete command?

Reply:
sure will add confirmation

Reply:
added confirmation